pulsar/consumer_test.go

```diff
@@ -5160,3 +5161,62 @@ func TestSelectConnectionForSameConsumer(t *testing.T) {
 			"The consumer uses a different connection when reconnecting")
 	}
 }
+
+func TestAckIDListWillIncreaseAckCounterMetrics(t *testing.T) {
```

Comment:
Could you reuse the existing `TestAckIDList`? This test has covered more cases and you won't need to write duplicated code (send and ack).